docs/data/date-pickers/date-field/CustomUIDateField.tsx

```diff
@@ -15,21 +11,6 @@ import {
   UseDateFieldComponentProps,
 } from '@mui/x-date-pickers/DateField';
 
-type JoyDateFieldProps = UseDateFieldComponentProps<Dayjs, Dayjs, JoyTextFieldProps>;
-
-const JoyDateField = (props: JoyDateFieldProps) => {
-  const { inputRef, inputProps } = useDateField<Dayjs, Dayjs, JoyDateFieldProps>(
-    props,
-  );
-
-  return (
-    <JoyTextField
```

Comment:
@flaviendelangle Are you fine with this?
It was causing issues after latest joy bump.
I noticed that we did already remove this case in `next`, hence it did not seem logical to try and migrate this case, especially when this page/demo is not listed anywhere. 🙈

Reply:
Sure